test/repository_test.rb

```diff
@@ -4,8 +4,9 @@
   let(:user1) { User.new(name: 'L') }
   let(:user2) { User.new(name: 'MG') }
   let(:users) { [user1, user2] }
+  let(:category) { Category.new }
 
-  let(:article1) { Article.new(user_id: user1.id, title: 'Introducing Lotus::Model', comments_count: '23') }
+  let(:article1) { Article.new(category_id: category.id, user_id: user1.id, title: 'Introducing Lotus::Model', comments_count: '23') }
```

Comment:
be careful with this because ID is nil, maybe you could pass an id to `Category.new`